settings/docker.yml

```diff
@@ -0,0 +1,7 @@
+aws_iterations_bucket: exercism-staging-iterations
+dynamodb_tooling_jobs_table: tooling_jobs
+tooling_orchestrator_url: http://tooling-orchestrator:3021
+anycable_redis_url: redis://redis:6379/1
+anycable_rpc_host: 0.0.0.0:50051
+rds_master_endpoint: mysql
+rds_port: 3306
```

Comment:
@erikschierboom: The settings for development, docker, and CI (GHA) now all live in these files. They are loaded into dynamodb by the bin script bundled with this gem. So in any project that has this in its Gemfile, you can just run `setup_exercism_config` and it will load the relevant settings into dynamo. You'll see this script is now listed in the github workflows and the dockerfiles of the other projects.